pkg/virt-handler/mount-manager/container-disk/mount.go

```diff
@@ -596,14 +434,6 @@ func (m *mounter) unmountKernelArtifacts(vmi *v1.VirtualMachineInstance) error {
 			// cleaning the mounted files.
 			log.Log.Object(vmi).Reason(err).Error("unable to unmount kernel artifacts")
 		}
-
-		removeSliceElement := func(s []vmiMountTargetEntry, idxToRemove int) []vmiMountTargetEntry {
```

Comment:
Is this dead code or why do we remove it?

Reply:
Good catch! I thought it was enough to remove entirely the vmi but we might have failures before we reach the place where we definitely remove the record entry